### PR TITLE
Product page cart page and some  bugs fixed 

### PR DESCRIPTION
**Pull request by tushitshukla03**
## Overview

1. This PR fixes or fixes part of #[fill_in_number_here].
2. This PR does the following: [Explain here what your PR does and why]

## Essential Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)
- [ ] The PR is made from a branch that's **not** called "main/master".

## PR Pointers

- If you need a review or an answer to a question, and don't have permissions to assign people, **leave a comment** like the following: "{{Question/comment}} @{{reviewer_username}} PTAL".
- Never force push. If you do, your PR will be closed.

### CODE REVIEW

File: frontend/pages/checkOut.vue [hunk 1]
```diff
@@ -173,32 +185,42 @@ export default {
...
       e.preventDefault();
       this.loading = true;
       let email = this.$store.getters['auth/userEmail'];
       console.log("email", email)
+      console.log(this.formValue)
       const data = {
         "first_name": this.formValue.firstName,
         "last_name": this.formValue.lastName,
-        "email": "priyanshmehta61@gmail.com",
+        "email": this.formValue.email,
```
Comment (Priyansh61) [inline frontend/pages/checkOut.vue:212]: yaha pai jo email id aati hai wo login user hona chahiye else backend will give errors.

File: frontend/pages/checkOut.vue [hunk 2]
```diff
@@ -173,32 +185,42 @@ export default {
...
         "hostel_address": this.formValue.hostel_address,
         "roll": this.formValue.roll,
         "branch": this.formValue.branch,
         "club_member": this.formValue.club_member,
         "phone": this.formValue.phone,
         "consent": this.formValue.consent,
-        "comment": this.formValue.comment,
-        "price": this.price,
+        "comment": "fdsdf",
```
Comment (Priyansh61) [inline frontend/pages/checkOut.vue:219]: Why arent we including any comments here?

File: frontend/pages/checkOut.vue [hunk 3]
```diff
@@ -173,32 +185,42 @@ export default {
...
+        "price": this.$store.getters['getPrice'],
         "prod_name": "needs to be added dynamically",
```
Comment (Priyansh61) [inline frontend/pages/checkOut.vue:221]: kindly add the `prod_name`

File: frontend/pages/checkOut.vue [hunk 4]
```diff
@@ -173,32 +185,42 @@ export default {
...
         "prod_type": "ticket",
-        "prod_id": "1", // either the event id or the product id not the ticket id
+        "prod_id": "2", // either the event id or the product id not the ticket id
```
Comment (Priyansh61) [inline frontend/pages/checkOut.vue:223]: No `prod_id` given here, pass this here from the frontend

**Review by Priyansh61 — CHANGES_REQUESTED** [review]
@tushitshukla03 kindly take a look.

File: frontend/pages/checkOut.vue (hunk 2 above)
Comment (tushitshukla03) [reply frontend/pages/checkOut.vue:219]: removed it was unnecessary

File: frontend/pages/checkOut.vue (hunk 4 above)
Comment (tushitshukla03) [reply frontend/pages/checkOut.vue:247]: I don't know as initially only one product will be there and how can I pass it from frontend, isn't prod_id should be unique and linked to product in backend??

File: frontend/pages/checkOut.vue (hunk 3 above)
Comment (tushitshukla03) [reply frontend/pages/checkOut.vue]: Done

**Pull request merged**